### DISCUSSION TITLE
Cannot connect to the secured Confluent SchemaRegistry instance with required key/cert authentication.

### DISCUSSION THREAD

**Discussion by sm-shevchenko** [provectus/kafka-ui#3460] [Q&A] [answered]
I need to connect KafkaUI to the secured Confluent SchemaRegistry instance with required key/cert authentication.
With CURL i can do it successfully using "_--key_" and  "_--cert_" parameters. Our SchemaRegistry configuration file has "ssl.client.authentication=_REQUIRED_" parameter. When i change config to "ssl.client.authentication=_REQUESTED_" KafkaUI can connect to  SchemaRegistry.

Can You help me please?

**Comment by Haarolean** [comment, marked as answer]
https://github.com/provectus/kafka-ui/blob/master/documentation/compose/kafka-ssl-components.yaml#L24
